__tests__/index.test.ts

```diff
@@ -108,6 +109,42 @@ describe('cli', () => {
     versionMock.done();
   });
 
+  describe('logged-in user notifications', () => {
+    let consoleInfoSpy;
+    const getCommandOutput = () => {
+      return [consoleInfoSpy.mock.calls.join('\n\n')].filter(Boolean).join('\n\n');
+    };
+
+    beforeEach(() => {
+      conf.set('email', 'owlbert@readme.io');
+      conf.set('project', 'owlbert');
+      conf.set('apiKey', '123456');
+      consoleInfoSpy = jest.spyOn(console, 'info').mockImplementation();
+    });
+
+    afterEach(() => {
+      consoleInfoSpy.mockRestore();
+      conf.clear();
+    });
+
+    it('should inform a logged in user which project is being updated', async () => {
+      const openapi = new OpenAPICommand();
+      const key = '123456';
+
+      await expect(openapi.run({ key, create: true, update: true })).rejects.toStrictEqual(
+        new Error('The `--create` and `--update` options cannot be used simultaneously. Please use one or the other!')
+      );
```

Comment:
If my approach in this comment works and the error message is the same as that one, could you use that pattern instead of importing the `openapi.run` function here?